src/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2020
```

Comment:
what's this for?

Reply:
it may not be needed now that we support TS. but i needed the "Optional Chaining" feature https://www.freecodecamp.org/news/javascript-new-features-es2020/

I will try to remove it and see if everything still works